include/nbl/video/utilities/CDumbPresentationOracle.h

```diff
@@ -0,0 +1,92 @@
+// Copyright (C) 2018-2021 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef _NBL_VIDEO_UTILITIES_C_DUMB_PRESENTATION_ORACLE_H_INCLUDED_
+#define _NBL_VIDEO_UTILITIES_C_DUMB_PRESENTATION_ORACLE_H_INCLUDED_
+
+#include <nabla.h>
```

Comment:
no engine file shall include the top level engine header.

Reply:
resolved